topics/epigenetics/tutorials/cut_and_run/tutorial.md

```diff
@@ -571,9 +552,9 @@ We can further remove some noise with a positive control, that is why we have do
 > >
 > > > <solution-title></solution-title>
 > > >
-> > > 1. 2,409 peaks
-> > > 2. 454 peaks
-> > > 3. Our precision is ~ 84%. A high precision is an indicator that we can predict true binding regions with high confidence.
+> > > 1. 2,865 peaks (this is the number of lines of our last output)
+> > > 2. About 4000 peaks (Rep1 and Rep2 has about 6-7 thousand peaks each)
```

Comment:
@heylf  Here I am not sure of what you consider false positive but I guess they are peaks in a single replicate which are not overlapping the second replicate.

Reply:
Maybe what I thought back then, true positive are peaks that are identified for GATA1 in both ChiP and CUT&RUN data. And false positives are peaks which were identified in CUT&RUN but do not occur in the ChIP seq data.

Reply:
Oh yes sorry I mixed the 2 sections. I will push a fix.